README-en.md

```diff
@@ -0,0 +1,172 @@
+* If you have a technical question, please try and figure it out before posting here. If you're having trouble, all questions are welcome.
+* Before asking any questions, please check if there's a channel for it before asking in #android-general
+* Job posting can be posted in #hiring only. Do not reach out to individuals for hiring purposes.
+
+**If you experience abuse, harassment, discrimination, or feel unsafe, let a moderator know. Here is a list of the current admins and their Slack IDs:**
+
+*The role of the admins is to be an unbiased mediator, they will not moderate or edit anything written in the Slack unless it is required as a result of a discussed dispute.*
+
+## Member Nomination  
+
+To nominate someone to join the Android Study Group, log in the [ASG web page](https://asgweb.herokuapp.com/) with your Android Study Group slack account, go in the **Nominate** page, and fill in the form with your name, their name and info, and why you are nominating them. You'll need to have been a member of Android Study Group for 30 or more days to be able to nominate someone. 
+
+This study group is for people actively involved in creating Android applications professionally. This includes people like software engineers and designers, and excludes people like journalists or recruiters. The admins will take into consideration the amount of time the nominee has spent working on Android (at least 1 year is ideal). Other helpful data points include the company where they work (if applicable), apps they have built, and their contributions to the Android community (via open source software, meetups, blog posts, etc). Membership is determined by a vote from the admins, who will then send out an invitation to the nominee.
+
+If there is any person you would like to keep out of the study group for any reason, message one of the admins.  
+
+Because this Slack group is representative of the community, admins reserve the right to deny admission or revoke membership based on CoC violating behavior outside of the Slack chat environment (e.g. in person, GitHub, social media), if necessary.
+
+## Good manners - Slack
+
+## Slack Commands
+
+### Commands
+
+Many members of this group use Slack at work, or have it installed on their cell phone. Therefore, the @channel and @everyone commands have been disabled for normal users and are only used by moderators only when needed.
+
+If in doubt about using Slack or some specific command, look for an admin.
+
+### Questions
+
+Before asking a question, take the time and search Google as well as the Android documentation for your question. If after that the answer is not clear to you, please refer to the sources you consulted to answer your question. Sending your question to the most appropriate channel can also facilitate discussion and resolution.
+
+When asking and answering questions, avoid any imperative or impatient language, and be cordial, after all we are all here to help one another!
```

Comment:
Aqui acho que `here to help each other` ficaria melhor.

Reply:
_When asking and answering questions, avoid any imperative or impatient language, and be cordial, after all we here are just trying to help each other._